layouts/partials/header.html

```diff
@@ -82,9 +81,10 @@ <h3 class="self-intro">{{ .Site.Params.selfintro }}</h3>
             <a href="mailto:{{ .Site.Params.email }}"><img class="icon" src="/img/email.svg" alt="email" /></a>
         {{ end }}
 
-        {{ if .Site.RSSLink }}
```

Comment:
https://gohugo.io/templates/rss/#reference-your-rss-feed-in-head uses `.RSSLink` (not .Site.RSSLink)